dia/attribution2.rkt

```diff
@@ -0,0 +1,62 @@
+#lang racket
+
+(provide (rename-out [mb #%module-begin]) #%datum #%top #%top-interaction #%app)
+
+#| (Unstable) Module language for attributions
+
+Format:
+  #lang abe/attribution2 exported-attributions-id
```

Comment:
nit: It could be more clear to explicitly declare what these expressions are, something like:

```
#lang abe/attributions

(export export-id)
```
Not sure if that would inordinately complicate the implementation?

Reply:
I would have to change the (syntax) readers; the result of reading modules would probably look the same (and thus have the same expansion).

I suppose I was trying to keep the metadata out of the way at the top; it's not strictly necessary to keep it on the same line. But I'm happy to try some different syntaxes. I won't have time to update the PR today, so let me know what syntax you'd like for the imports and exports. The more Racket-y (S-expressions) the easier it will be to read :) I'm already using Racket's readers to grab these first metadata bits.

Reply:
Ok, thanks for explaining the considerations. I think it's fine as is. My concern initially was that it seemed a little bit magical / opaque, but if we want to keep the metadata in the black box for now, I think that works.